src/plugins/intel_gna/src/layers/gna_layer_info.hpp

```diff
@@ -349,6 +349,12 @@ class LayerInfo {
     bool isNonValuesChangable() const {
         return isNonFunctional() || isSplit() || isSlice() || isConcat();
     }
+    bool is_gather() const noexcept {
+        return isOfType("gather");
+    }
+    bool is_fq_non_sensitive() const noexcept {
```

Comment:
Shouldn't we use isNonFunctional instead and just extend it?

Reply:
gather and permute are not non-functional layers, it is better to isolate them in separate condition

Reply:
And isNonFunctional() check a bit complicated and we will not call it at all in the most cases in is_fq_non_sensitive()